### PR TITLE
[BE-2588]:Added default version for default workflows

### PR DESCRIPTION
**Pull request by boztopuz**

### CODE REVIEW

**Review by csonuryilmaz — CHANGES_REQUESTED** [review]
@boztopuz React Native has also "Cocoapods Install" step in default workflow.

**Comment by boztopuz** [comment]
> @boztopuz React Native iOS has also "Cocoapods Install" step in default workflow.

I didn't fix it on purpose, because I didn't want to forget a question I had to ask. In the default workflows there are commit IDs for components, and I couldn't understand what these IDs are based on. For example, the commit ID of a default component in version 2.0.* comes from version 1.0.0.

**Comment by csonuryilmaz** [comment]
@boztopuz 
> In the default workflows there are commit IDs for components, and I couldn't understand what these IDs are based on.

I investigated the `commit` in the YAML file both from Git history and actual testing.

TLDR; the `commit` is unused actually. **componentVersion** is actively used.

I tested "Increment Build and Version Number" with a fresh build profile. (default workflows)

YAML :point_down: 

```yaml
- componentType: appcircle_ios_build_version_increment
  componentVersion: 2.0.*
  stepName: Increment Build and Version Number
  webUrl: 'https://github.com/appcircleio/appcircle-ios-build-version-increment'
  repoUrl: 'https://github.com/appcircleio/appcircle-ios-build-version-increment.git'
  commit: 4707781
  enabled: true
  onError: $exit
  alwaysRun: false
...
```

When we search for the commit `4707781` we could not find a valid version. :x:

CWD: appcircle-workflow-components/appcircle_ios_build_version_increment
```bash
bsh ❯ grep -nri '4707781' .
```

On UI we can see that the selected version is `2.0.x` as defined in YAML. :heavy_check_mark: 

When we run the build  pipeline without any workflow modification, the step commit is `a14f5dd`. It's the latest version as expected.

```bash
bsh ❯ grep -nri 'a14f5dd' .
./2.0.3/component.yaml:11:commit: a14f5dd
```

I exported the default workflow (download) and replaced all `commit` values with empty value in the YAML.
- When I upload the cleaned up YAML as a new workflow, it's success. :heavy_check_mark: 
- When I run the pipeline with clean workflow, it's worked as before. :heavy_check_mark: 

TLDR; the `commit` in the default workflows YAML has no effect on build pipeline.  **componentVersion** is important.

---

:memo:  _Note:_ I will add a refactoring issue to our backlog for removing the `commit` from YAML.

**Comment by boztopuz** [comment]
@csonuryilmaz I understand properly, thanks your time and interest. BTW, I changed react side as well.
